src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -242,3 +251,19 @@ def load_arguments(self, _):
 
     with self.argument_context('spring-cloud app custom-domain update') as c:
         c.argument('certificate', help='Certificate name in Azure Spring Cloud.')
+
+    with self.argument_context('spring-cloud app-insights update') as c:
+        c.argument('app_insights_key',
+                   help="Instrumentation key of the existing Application Insights",
+                   validator=validate_app_insights_parameters)
+        c.argument('app_insights',
+                   help="Name of the existing Application Insights in the same Resource Group. Or Resource ID of the existing Application Insights in a different Resource Group.",
+                   validator=validate_app_insights_parameters)
+        c.argument('sampling_rate',
+                   type=float,
+                   help="Sampling Rate of application insights. Maximum is 100.",
+                   validator=validate_app_insights_parameters)
+        c.argument('disable',
```

Comment:
the parameter name should align with `disable_app_insights` in `create` command if they are doing the same thing

Reply:
ok, they are different things. disable_app_insights is the old feature but this sub command is to update new feature.